src/components/Avatar/avatar.css

```diff
@@ -24,7 +24,7 @@
   height: 24px;
   width: 24px;
 }
-.y-avatar__size-xSmall > .ms-Persona {
+.y-avatar__size-xSmall .ms-Persona {
```

Comment:
Basically Fabric does not allow avatars to be smaller than 32px, and this is our way to fix that. I think that the proper fix might be in Fabric. 

Also we can change the comment "extract a Fabric Avatar component out of Persona" since we never plan to do that.

Reply:
feel free to look at a fix within Fabric but it's not high priority, this style override is protected by visual diffs

extracting Avatar from Persona is still a viable project at some point -- it would reduce our amount of dependency JS from Fabric and other teams would benefit. also not high priority for us though